tests/unit/amo/components/Errors/TestErrorComponent.js

```diff
@@ -0,0 +1,61 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import NestedStatus from 'react-nested-status';
+
+import Card from 'ui/components/Card';
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+
+describe(__filename, () => {
+  function render(props = {}) {
+    return shallow(<ErrorComponent {...props} />);
+  }
+
+  it('renders an ErrorComponent', () => {
+    const root = render();
+
+    expect(root.find(NestedStatus)).toHaveProp('code', 400);
+    expect(root.find(Card)).toHaveClassName('Errors');
+  });
+
+  it('renders header when header is available', () => {
+    const root = render({ header: 'foo' });
```

Comment:
A better approach, which removes the possibility of a typo in a test, is to use a variable to set a value and then use the same variable in the assert. In this example, you could do:

```
const header = 'foo';
const root = render({ header });

expect(root.find(Card)).toHaveProp('header', header);
```